### PR TITLE
Remove without_storage_info for the amendments pallet

### PR DESCRIPTION
**Pull request by rajesh-nodle**
Part of [#173](https://github.com/NodleCode/chain-workspace/issues/173)

**HighLights**

- Not much changes, removed `without_storage_info` from pallet

### CODE REVIEW

File: pallets/amendments/src/lib.rs [hunk 1]
```diff
@@ -83,12 +82,12 @@ pub mod pallet {
...
-				T::WeightInfo::propose(
-					amendment.using_encoded(|x| x.len()) as u32,
-				).saturating_add(amendment.get_dispatch_info().weight),
-				DispatchClass::Operational,
+		#[pallet::weight((
+			T::WeightInfo::propose(
+				amendment
+					.using_encoded(|x| x.len()) as u32)
+					.saturating_add(amendment.get_dispatch_info().weight),
+					DispatchClass::Operational,
```
Comment (aliXsed) [inline pallets/amendments/src/lib.rs:90]: Can you please explain where this change has come from and how it's related to to without_storage_info?

Comment (rajesh-nodle) [reply pallets/amendments/src/lib.rs:90]: looks like from `cargo fmt`, let me recheck this

Comment (aliXsed) [reply pallets/amendments/src/lib.rs:90]: This doesn't look just formatting to me. The result of `propose(amendment)+amendment.weight` can be different from `propose(amendment + amendment.weight)`.

Comment (rajesh-nodle) [reply pallets/amendments/src/lib.rs]: You are right, Done

**Comment by ETeissonniere** [comment]
@rajesh-nodle @aliXsed let's close. I am removing `amendments` as it isn't used anymore.

**Pull request closed without merging**